openlibrary/plugins/openlibrary/lists.py

```diff
@@ -214,10 +214,24 @@ def is_enabled(self):
 
     def GET(self, path):
         # If logged in patron is viewing their lists page, use MyBooksTemplate
-        username = path.split('/')[-1]
-        mb = MyBooksTemplate(username, 'lists')
-        template = self.render_template(path, show_header=False)
-        return mb.render(template=template, header_title=_("Lists"), page=mb.user)
+        if path.startswith("/people/"):
```

Comment:
I wonder if we should simply break the `path` regex into different functions rather than trying to collapse them into one, just a suggestion.